ndk-make.sh

```diff
@@ -10,9 +10,9 @@
 #
 #     adb shell uname -m
 #
-# or:
-#
-#     adb shell cat /proc/cpuinfo
```

Comment:
`adb shell cat /proc/cpuinfo` creates a very long output, which makes it a lot less helpful.